src/app/service/poll.service.ts

```diff
@@ -19,18 +20,21 @@ export class PollService {
   /**
    * Initialize the service
    * @param http Injected http client
+   * @param tools Injected CommonToolsService
    */
   constructor(
-    private http: HttpClient
-  ) {}
+    private http: HttpClient,
+    private tools: CommonToolsService
+  ) {
+  }
 
   /**
    * Creates a poll on the server
    *
    * @param poll Affected poll
    */
   create(poll: Poll): Observable<Poll> {
-    return this.http.post<Poll>(ENDPOINT_URL, poll, { withCredentials: true });
+    return this.http.post<Poll>(ENDPOINT_URL, poll, {withCredentials: true});
```

Comment:
wait, the linter removed the whitespaces here?

Reply:
same for other lines in this file

Reply:
Yes, seems like

Reply:
ok